### PR TITLE
Align buttons in info-grid

### PR DESCRIPTION
**Pull request by mrleewong**

### CODE REVIEW

File: src/scss/_info-grid.scss [hunk 1]
```diff
@@ -3,28 +3,29 @@ $m: 'info-grid';
...
-
 		&-icon {
 			position: relative;
 			width: 120px;
 			height: 120px;
 			border-radius: 50%;
 			// border: 1px solid rgba($color-gray-dark, 0.1);
 			background-color: rgba($color-white, 0.5);
-			margin-bottom: 40px;
+			margin-bottom: 20px;
```
Comment (stpdenis) [inline src/scss/_info-grid.scss]: You use pixels and em, can you convert it into em/rem?

File: get-dash/index.html [hunk 2]
```diff
@@ -332,47 +332,49 @@ <h2 class="section__title">Use Dash everywhere</h2>
...
 						</div>
 						<h3 class="info-grid__item-title">{% t pages.get-dash.visa-heading %}</h3>
-						<p class="info-grid__item-text">{% t pages.get-dash.visa-text %}</p>
+						<span class="info-grid__item-text">{% t pages.get-dash.visa-text %}</span>
```
Comment (stpdenis) [inline get-dash/index.html]: The file is massive, should be split into partials

Comment (stpdenis) [reply get-dash/index.html]: Maybe not as part of this PR

Comment (mrleewong) [reply get-dash/index.html]: @stpdenis maybe next time, not urgent.

The rest is fixed

**Pull request merged**